nautobot/docs/user-guides/graphql.md

```diff
@@ -11,10 +11,10 @@
 When interacting with APIs, It's often necessary to build relationships between multiple models to achieve the result that is desired. Doing this typically requires multiple API calls to create the relationships. For example, lets assume that there are two devices in Nautobot. Each are assigned a site, region, roles, interfaces, and IP Addresses.
 
 Simply querying the `/api/dcim/devices/` API route provides:
-<div>
-  <details>
-    <summary>View API Results</summary>
-```
+
+#### View API Results
```

Comment:
The `<details>` sections were added intentionally to collapse by default the rather lengthy examples. Why remove them?